docs/UserGuide.md

```diff
@@ -940,8 +940,11 @@ Parameters      |  Explanation
 
 <div markdown="span" class="alert alert-warning">:exclamation: **Note:**
```

Comment:
oh the error because if wanna use markdown, use markdown="block" not span

Reply:
ahh right ok lemme update that